style.css

```diff
@@ -27,54 +27,56 @@ body {
 ================================*/
 form {
   background: #2c3e50;
-  height: 2.3em; 
+  height: 2.3em;
   width: 100%;
+  display: flex;
```

Comment:
Correct, I changed the message input form to flexbox because it was the quickest and easiest way to fix the mobile layout.  However, flexbox doesn't play nice with older IE browsers (below IE 11), so if a more universal solution is necessary I'd be happy to make that change.

While on the subject, are there any thoughts on which browsers should be supported by this project?